### ISSUE TITLE
Tiny-Small networks load failure & training issue

### ISSUE THREAD

**Issue by Sbrunoberenguel** [jamycheung/Trans4PASS#7]
Hello,
thanks a lot for providing the code of your article.
I'm working on something similar and I'm trying to make a comparison with your work. When I try to evaluate on the Stanford2D3DS dataset the error attached appears. It only appears on the Trans4PASS network small and tiny (not in Trans4PASS+). As I guess, is a failure when loading the network's weights, so the results are far from reliable (they look random, with a mIoU=2.7)
Have you any insight of what may be happening here? 

Also, I re-trained the "tiny" network from scratch on the Stanford dataset and some other problem may appear since the results I get are far from the ones you provide (also, my resources are limited). I'm also attaching the log of the training. May you guess why there is such a difference in performance?

Thanks

[eval_error.txt](https://github.com/jamycheung/Trans4PASS/files/10866810/eval_error.txt)
[2023-02-24-06-40_Trans4PASS_trans4pass_v1_stanford2d3d_pan_log.txt](https://github.com/jamycheung/Trans4PASS/files/10866852/2023-02-24-06-40_Trans4PASS_trans4pass_v1_stanford2d3d_pan_log.txt)

**Comment by jamycheung** [comment]
Hi,
thanks for your interest. About loading the model weight, you can modify the DMLP module for the Trans4PASS version. Please check [here](https://github.com/jamycheung/Trans4PASS/blob/main/segmentron/models/trans4pass.py#L5-L8). 
About the training issue, the batch size is an important factor. If to train on limited resource, you can check and try gradient accumulation. We did not test it with this. It might help.
